pkg/dispatcher/config.go

```diff
@@ -128,7 +130,7 @@ func (config *Config) DetermineClusterForJob(jobBase prowconfig.JobBase, path st
 	}
 
 	if config.DetermineE2EByJob {
-		if cloud := DetermineCloud(jobBase); cloud != "" {
+		if cloud := config.DetermineCloudMapping(jobBase); cloud != "" {
```

Comment:
Why does it have effect only when `config.DetermineE2EByJob=true`?

Reply:
and of course it's only place where I spotted we care about cloud determination which is a base for this change